Antonio_Ramos_Documentation

```diff
@@ -369,3 +369,15 @@ Tab2 = Tab2.astype(convert_dict)
 - changed pathway reference in Local_3Stage_kfunction.py to import new shapefile CA_ZCTA_Man2.0.shp
 
 - ran program Local_3Stage_kfunction.py for 2 simulations and it successfully saved a shapefile "Manufa_Emp_C000_Points.shp". Infinite simulation loop error now solved. 
+
+7/28/2020
+- ran program successfully for 499 simulations. Shapefile exported. However, there is an issue with the GoodLabs_New.shp file where the labs are not projecting correctly in ArcGis. Problem corrected by using GoodLabs_Project.shp instead. 
+
+7/30/2020
+- ran program successfully for 499 simulations, importing GoodLabs_Project.shp. However, this means that the code producing the labs point shapefile is not working properly and needs to be reviewed. 
```

Comment:
I think it's likely that the cluster file is not being produced because the significance levels are not high enough to create a cluster. If you don't know how to interpret the data in the point file to see whether this is the case or not, let me know and we can go through it together, or you can send me the output file and I'll check it out and let you know what I find.